<?xml version="1.0"?>
<launch>

	<rosparam file="$(find cob_controller_configuration_gazebo)/ros/controller/torso_v1_controller.yaml" command="load" />

	<node name="torso_controllers_spawner" pkg="controller_manager" type="spawner" args="torso_controller" />

</launch>
